media-sound/pnmixer/pnmixer-0.6_pre20111213.ebuild

```diff
@@ -1,7 +1,7 @@
-# Copyright 1999-2014 Gentoo Foundation
+# Copyright 1999-2018 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
+EAPI=6
```

Comment:
Please don't change EAPI of stable ebuilds. Only bump the newest version, and if it's stable (or EAPI change is likely to break stuff), then you need to create a revision bump.